evap/results/views.py

```diff
@@ -107,10 +114,15 @@ def course_detail(request, semester_id, course_id):
                                             {'total_votes': 0, 'sections': []})['sections'].append(section)
 
             # Sum up all Sections for this contributor.
-            # If section is not a RatingResult:
-            # Add 1 as we assume it is a TextResult or something similar that should be displayed.
-            contributor_sections[section.contributor]['total_votes'] +=\
-                sum([s.total_count if isinstance(s, RatingResult) else 1 for s in section.results])
+            for s in section.results:
+                if isinstance(s, TextResult):
+                    # If section is a TextResult:
```

Comment:
`s` is not a section here. Maybe remove the comment completely? I don't feel like these comments add any valuable information. I also feel like we should replace the variable `s` with something more descriptive like `result`